CMakeLists.txt

```diff
@@ -315,6 +323,7 @@ target_link_libraries(Test_ICP
   gtest
   gtest_main
 )
+target_compile_options(Test_ICP PRIVATE -g)
```

Comment:
This should probably not be set by default, but rather by CMAKE_BUILD_TYPE?

Reply:
Sorry about that, that one slipped through when I solved the memory alignment error that occured with Test_ICP. I'll remove it.